packages/superset-ui-legacy-preset-chart-deckgl/src/DeckGLContainer.jsx

```diff
@@ -88,31 +91,38 @@ export default class DeckGLContainer extends React.Component {
     return this.props.layers;
   }
 
+  setTooltip = tooltip => {
```

Comment:
do other components call these `setTooltip` `fn()`s? can't find references in the file so wanted to verify

Reply:
Yes, `AnimatedDeckGLContainer` and `CategoricalDeckGLContainer` contain `DeckGLContainer` and call it via `ref`.